flow/workflows/cdc_flow.go

```diff
@@ -152,6 +152,29 @@ func (w *CDCFlowWorkflowExecution) processCDCFlowConfigUpdates(ctx workflow.Cont
 			continue
 		}
 
+		ensurePullabilityCtx := workflow.WithActivityOptions(ctx, workflow.ActivityOptions{
+			StartToCloseTimeout: 5 * time.Minute,
+		})
+		ensurePullabilityFuture := workflow.ExecuteActivity(
+			ensurePullabilityCtx,
+			flowable.EnsurePullability,
+			&protos.EnsurePullabilityBatchInput{
+				PeerConnectionConfig: cfg.Source,
+				FlowJobName:          cfg.FlowJobName,
+				SourceTableIdentifiers: func() []string {
+					additionalSourceTables := make([]string, 0, len(flowConfigUpdate.AdditionalTables))
+					for _, additionalSourceTable := range flowConfigUpdate.AdditionalTables {
+						additionalSourceTables = append(additionalSourceTables, additionalSourceTable.SourceTableIdentifier)
+					}
+					return additionalSourceTables
+				}(),
+				CheckConstraints: true,
+			})
+		if err := ensurePullabilityFuture.Get(ctx, nil); err != nil {
+			w.logger.Error("failed to ensure pullability for additional tables: ", err)
+			return err
```

Comment:
Can this be removed?

Reply:
`processCDCFlowConfigUpdates` return value should not be aborting whole workflow. Any flow config update error should result in an error being reported to user in some way, & then being ignored. Then this can remain `return err`